custom/icds/messaging/indicators.py

```diff
@@ -121,6 +126,11 @@ def location_names_from_user_id(self, user_ids):
         return {self.awc_locations[l] for l in loc_ids}
 
 
+class AWWAggregatePerformanceIndicator(AWWIndicator):
+    def get_messages(self, language_code=None):
+        return LSAggregatePerformanceIndicator(self.domain, self.supervisor).get_messages(language_code)
```

Comment:
i'm not sure this is right - this will send to this one AWW the aggregated info for all AWCs under the LS right? we'd only want the AWW's info to be included here